message_definitions/v1.0/messages.xml

```diff
@@ -368,7 +368,17 @@
       <field name="alt" type="float" unit="m"/>
     </message>
 
-    <!-- 45 is free -->
+    <message name="CHIRP" id="45">
+      <field name="chirp_active"        type="uint8">Chirp is active</field>
```

Comment:
a little redundant to have chirp in the name of a variable in a chirp message, this isn't chirpception afterall.